beacon_chain/utils/shuffling/src/lib.rs

```diff
@@ -25,9 +25,15 @@ pub fn shuffle<T>(
     -> Result<Vec<T>, ShuffleErr>
 {
     let mut rng = ShuffleRng::new(seed);
+
     if list.len() > rng.rand_max as usize {
         return Err(ShuffleErr::ExceedsListLength);
     }
+
+    if list.len() == 0 {
```

Comment:
Clippy gives the following error here:

```
help: using `is_empty` is more concise: `list.is_empty()`
```

It works either way, but we're intending to implement clippy in our build process so it would be best to change this to `list.is_empty()` :)